### DISCUSSION TITLE
Window focus changing

### DISCUSSION THREAD

**Discussion by mshzsh** [hyprwm/Hyprland#970] [Q&A] [answered]
![sr](https://user-images.githubusercontent.com/41986782/200360115-06394de9-f90d-4c8a-a8e3-586d662b89b2.gif)

When moving between workspaces, the window focus changes. Of course, this does not happen when we navigate from right to left in workspace!
Is there a solution to fix this?

**Comment by vaxerski** [comment, marked as answer]
gif is so small I can't see shit. Post an mp4

**Reply by mshzsh** [reply]
https://user-images.githubusercontent.com/41986782/200522490-e347de51-8fcb-4cf9-af82-f13c559a2a14.mp4

**Reply by mshzsh** [reply]
any idea? :thinking:

**Reply by vaxerski** [reply]
odd. Can't reproduce.

**Reply by mshzsh** [reply]
I think you don't understand what I mean, the problem is that when navigate between workspaces, the windows focus changes, which is really annoying at times, do you have a plan to fix this problem?

**Reply by vaxerski** [reply]
I understand, but it doesn't happen to me

**Reply by mshzsh** [reply]
Can you please test with my config?
[hyprland.txt](https://github.com/hyprwm/Hyprland/files/10049970/hyprland.txt)

**Reply by mshzsh** [reply]
It works fine with mouse but not with keyboard

**Reply by mshzsh** [reply]
I fixed the problem! Or there was a mistake in the config file, thank you
